### PR TITLE
Vivian Lei hw01 cfss-sp 20

### PR DESCRIPTION
**Pull request by VivianLei0504**
This is Vivian Lei from the class, and here are some of my reflections for this assignment.

I think it was not too challenging for me, and I was able to get it done even before the second session of our class. The instructions on the homework page really helped me understand the workflow of this assignment, and by following the steps described in the video, I was able to get things done with ease. I also found the introduction to R-Markdown syntax on Github pretty helpful that one read-through of the page solved all the questions I had on my mind. However, there is still some space for improvement. I pushed "commit" too many times because I was experimenting with RStudio, but it was unnecessary, and too many "commits" would make the process less succinct & harder for another reader to tell which changes are necessary/unnecessary. I think this could improve with more practice and time. I also had some difficulties with adjusting the paragraph styles of the README file, but I was eventually able to figure everything out myself with the help of all the resources I had at hand.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,24 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# A Short Introduction of Myself
+Hello, this is **Vivian Lei**.
+I am a second-year student at the college, majoring in *English Language & Literature* and *Sociology*.
```
Comment (deblnia) [inline README.md:5]: 💯 major combo imho

File: README.md [hunk 2]
```diff
@@ -1 +1,24 @@
...
+
++ Cats
++ Red pandas
++ Penguins
+
+If you'd like to connect with me, you can send me a friend request on [Facebook](https://www.facebook.com/VivianLei1999)
+
+![a photo of my cat ](90217351_511233089587711_8380126470382551040_n.jpg)
+
+Finally, here is a photo of my cat Godot.
```
Comment (deblnia) [inline README.md:24]: As in _Waiting for_???

**Review by deblnia — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |    X       |              |            |
| **Coding strategy**         |   X        |              |            |
| **Presentation**            |       X    |              |            |
| **Achievement, creativity** |   X        |              |            |
| **Reproducibility**         |       X    |              |            |

Remarks:

* Great work Vivian! You successfully used Git and Github, and five elements of markdown syntax. 
* Good work on the images! Those can be tricky.

File: README.md (hunk 2 above)
Comment (wearnomask) [reply README.md:24]: Love your cat's name!!

File: README.md [hunk 3]
```diff
@@ -1 +1,24 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# A Short Introduction of Myself
+Hello, this is **Vivian Lei**.
+I am a second-year student at the college, majoring in *English Language & Literature* and *Sociology*. 
+I am from Yunnan Province, China, but I currently live in Chicago.
```
Comment (wearnomask) [inline README.md:6]: Yunnan is such a beautiful place, I (a Hunanese) lived there for a month a few years ago. Miss it so much!

**Review by jaycgibbs — COMMENTED** [review]
Also a big fan of the cat. Not just the name but really everything tbh

**Comment by wearnomask** [comment]
I don't have any critical feedback on your work. You successfully included five different markdown elements perfectly (header, bold & italic, link, image, and two lists), and you put the image in the right place. Also, thanks for sharing your experience in detail in your report. I can definitely relate to those struggles.

**Comment by jaycgibbs** [comment]
I also don't have any critical feedback. Anything I could really think of would just be pedantic and probably not worth saying just to say. The headers are nicely done, the lists are helpful, and the image is as properly used as it is cool to look at.

**Pull request closed without merging**